src/v/utils/tests/BUILD

```diff
@@ -505,3 +505,22 @@ redpanda_cc_bench(
         "@seastar//:benchmark",
     ],
 )
+
+redpanda_cc_btest(
+    name = "external_process_test",
+    timeout = "short",
+    srcs = [
+        "external_process_test.cc",
+    ],
+    data = [":handle-sigterm.sh"],
+    env = {
+        "HANDLE_SIGTERM_SCRIPT": "$(location :handle-sigterm.sh)",
```

Comment:
location is soft deprecated. It's recommended to use `execroot` instead.

```
"HANDLE_SIGTERM_SCRIPT": "$(execroot :handle-sigterm.sh)",
```

Reply:
I replaced `location` with `execroot` but that didn't appear to return the path to handle-sigterm.